pages/common/tmux.md

```diff
@@ -1,7 +1,6 @@
 # tmux
 
-> Tmux is a terminal multiplexer. It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
-> By default, your tmux prefix is Ctrl-B.  When you see {{prefix}}, use Ctrl-B, if you haven't changed your tmux prefix. If you have, use your prefix.
+> Tmux is a terminal multiplexer. It allows you to have multiple sessions with windows, panes, and so much more!
```

Comment:
Perhaps this will read better?
```suggestion
> Terminal multiplexer. It allows for having multiple sessions with windows, panes, and more.
```

Just a little more formal to fit with other pages :-)